### PR TITLE
Adding new template for Alcatel aos: `show system` Co-authored-by: Michael Bear <38406045+mjbear@users.noreply.github.com>

### PR DESCRIPTION
**Pull request by evilmonkey19**

### CODE REVIEW

**Comment by mjbear** [comment]
@evilmonkey19
This PR also has multiple template changes.

**Comment by evilmonkey19** [comment]
Fixed ✅ It was not intended but rather that i'm doing local development.

File: ntc_templates/templates/index [hunk 1]
```diff
@@ -11,6 +11,7 @@
 #
 Template, Hostname, Platform, Command
 
+alcatel_aos_show_system.textfsm, .*, alcatel_aos, show system
```
Comment (mjbear) [inline ntc_templates/templates/index:14]: Does the AOS command line support any shorthand for `show system`?

File: ntc_templates/templates/alcatel_aos_show_system.textfsm [hunk 2]
```diff
@@ -0,0 +1,27 @@
+Value DESCRIPTION (.+,.+,[^,]+)
+Value OBJECT_ID ([^,]+)
+Value UP_TIME (\d+ days \d+ hours \d+ minutes and \d+ seconds)
+Value CONTACT (.+,(?: https://\S+)?)
+Value NAME ([^,]+)
+Value LOCATION ([^,]+)
+Value Service (\d+)
```
Comment (mjbear) [inline ntc_templates/templates/alcatel_aos_show_system.textfsm:7]:
```suggestion
Value SERVICES (\d+)
```

Make plural to match raw output.
Capitalize capture group name.

File: ntc_templates/templates/alcatel_aos_show_system.textfsm [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Value DESCRIPTION (.+,.+,[^,]+)
+Value OBJECT_ID ([^,]+)
+Value UP_TIME (\d+ days \d+ hours \d+ minutes and \d+ seconds)
+Value CONTACT (.+,(?: https://\S+)?)
+Value NAME ([^,]+)
+Value LOCATION ([^,]+)
```
Comment (mjbear) [inline ntc_templates/templates/alcatel_aos_show_system.textfsm:6]:
```suggestion
Value DESCRIPTION (.+)
Value OBJECT_ID (.+)
Value UP_TIME (.+)
Value CONTACT (.+)
Value NAME (.+)
Value LOCATION (.+)
```

Simplify regexes.
Simpler and flexible should mean less breakage over time.

File: ntc_templates/templates/alcatel_aos_show_system.textfsm [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+Value Service (\d+)
+Value DATE_TIME (\S+\s*\S+\s*\d+\s*\d+\s*\d+:\d+:\d+\s*(\(\w+\)))
+Value AVAILABLE (\d+)
+Value COMMENTS ([^,]+)
```
Comment (mjbear) [inline ntc_templates/templates/alcatel_aos_show_system.textfsm:10]:
```suggestion
Value COMMENTS (.+)
```

Simplify regex.

File: ntc_templates/templates/alcatel_aos_show_system.textfsm [hunk 5]
```diff
@@ -0,0 +1,27 @@
+Value DESCRIPTION (.+,.+,[^,]+)
+Value OBJECT_ID ([^,]+)
+Value UP_TIME (\d+ days \d+ hours \d+ minutes and \d+ seconds)
+Value CONTACT (.+,(?: https://\S+)?)
+Value NAME ([^,]+)
+Value LOCATION ([^,]+)
+Value Service (\d+)
+Value DATE_TIME (\S+\s*\S+\s*\d+\s*\d+\s*\d+:\d+:\d+\s*(\(\w+\)))
```
Comment (mjbear) [inline ntc_templates/templates/alcatel_aos_show_system.textfsm:8]:
```suggestion
Value DATE_TIME ((\S+\s*)+)
```

Simplify regex.

Fortunately you can (and are) anchoring the match line with that trailing comma. :grinning:

**Review by mjbear — CHANGES_REQUESTED** [review]

File: ntc_templates/templates/alcatel_aos_show_system.textfsm (hunk 2 above)
Comment (mjbear) [reply ntc_templates/templates/alcatel_aos_show_system.textfsm:7]: If this suggestion (and others) help, click the button here to accept them. Thank you!

File: ntc_templates/templates/alcatel_aos_show_system.textfsm [hunk 6]
```diff
@@ -0,0 +1,27 @@
+Value DESCRIPTION (.+,.+,[^,]+)
+Value OBJECT_ID ([^,]+)
+Value UP_TIME (\d+ days \d+ hours \d+ minutes and \d+ seconds)
```
Comment (mjbear) [inline ntc_templates/templates/alcatel_aos_show_system.textfsm:3]:
```suggestion
Value UPTIME (.+)
```

Move `UP_TIME` to `UPTIME` as that is a common capture group name. (We're hoping to standardize/normalize where we can.)

File: ntc_templates/templates/alcatel_aos_show_system.textfsm [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+Value Service (\d+)
+Value DATE_TIME (\S+\s*\S+\s*\d+\s*\d+\s*\d+:\d+:\d+\s*(\(\w+\)))
+Value AVAILABLE (\d+)
+Value COMMENTS ([^,]+)
+
+Start
+  ^\s*System:$$
+  ^\s*Description:\s*${DESCRIPTION},$$
+  ^\s*Object\s*ID:\s*${OBJECT_ID},$$
+  ^\s*Up\s*Time:\s*${UP_TIME},$$
```
Comment (mjbear) [inline ntc_templates/templates/alcatel_aos_show_system.textfsm:16]:
```suggestion
  ^\s*Up\s*Time:\s*${UPTIME},$$
```

Match capture group change (as noted above in the file) to the common occurrence.

File: ntc_templates/templates/alcatel_aos_show_system.textfsm [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+  ^\s*Contact:\s*${CONTACT},$$
+  ^\s*Name:\s*${NAME},$$
+  ^\s*Location:\s*${LOCATION},$$
+  ^\s*Services:\s*${Service},$$
```
Comment (mjbear) [inline ntc_templates/templates/alcatel_aos_show_system.textfsm:20]:
```suggestion
  ^\s*Services:\s*${SERVICES},$$
```

Change capture group name and make all uppercase.

File: ntc_templates/templates/alcatel_aos_show_system.textfsm (hunk 6 above)
Comment (mjbear) [reply ntc_templates/templates/alcatel_aos_show_system.textfsm:3]: https://github.com/networktocode/ntc-templates/blob/master/docs/dev/data_model.md

**Comment by evilmonkey19** [comment]
I added all your suggestions 😀 Thanks again! I added also the short-handed version

**Comment by mjbear** [comment]
> I added all your suggestions 😀 Thanks again! I added also the short-handed version

@evilmonkey19
Nice!

Was the [Commit suggestion button](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) available?
Based on the commit history it doesn't seem like it.
For local development, when you accept changes from GH, you'd run a `git pull` to pull down changes to be up-to-date on your local system.

Commit suggestion button

**Comment by evilmonkey19** [comment]
Yees it was available hehe. Simply just that sometimes i like to do things manually as everthing is automated :P

**Comment by mjbear** [comment]
> Yees it was available hehe. Simply just that sometimes i like to do things manually as everthing is automated :P

Darn.
We co-authored on this one and now it doesn't show it. :disappointed:

**Comment by evilmonkey19** [comment]
Is there anything i could do to add you?

**Comment by mjbear** [comment]
> Is there anything i could do to add you? Simply add a space or something similar and i add it :)

I'll see if there's anything else, but for now things appear to be fairly complete on this PR.
Though I did come up with a documentation idea.

If you were to add co-author text to the PR description (it might get picked up).
For example, GH auto-added similar text at [this previous commit for another PR](https://github.com/networktocode/ntc-templates/commit/c18104e983a60b5f7a7461def798fb84ab3e0d61).

```
Co-authored-by: Michael Bear <38406045+mjbear@users.noreply.github.com>
```

**Review by jmcgill298 — APPROVED** [review]

**Review by jmcgill298 — APPROVED** [review]

**Pull request merged**